platforms/aws/aws.go

```diff
@@ -0,0 +1,71 @@
+package aws
+
+import (
+	"encoding/json"
```

Comment:
Use bharyang :D

Reply:
Go formatter uses ascending order for imports  ;P

Reply:
Tara looks like first three are not following the format.

Reply:
ascending order by name, not by length as bharyang

Reply:
eh hehe.. did'nt understand the first time :D